crates/driver/src/tests/cases/protocol_fees.rs

```diff
@@ -142,6 +151,38 @@ async fn surplus_protocol_fee_buy_order_not_capped() {
             },
         },
         expected_score: 20.ether().into_wei(),
+        fee_handler: FeeHandler::Driver,
+    };
+
+    protocol_fee_test_case(test_case).await;
+}
+
+#[tokio::test]
+#[ignore]
+async fn protocol_fee_calculated_on_the_solver_side() {
+    let fee_policy = Policy::Surplus {
+        factor: 0.5,
+        max_volume_factor: 1.0,
+    };
+    let test_case = TestCase {
+        fee_policy,
+        order: Order {
+            sell_amount: 50.ether().into_wei(),
+            buy_amount: 40.ether().into_wei(),
+            side: order::Side::Buy,
+        },
+        execution: Execution {
+            solver: Amounts {
+                sell: 30.ether().into_wei(),
+                buy: 40.ether().into_wei(),
+            },
+            driver: Amounts {
+                sell: 40.ether().into_wei(),
+                buy: 40.ether().into_wei(),
+            },
```

Comment:
These values don't seem to matter at all for the test.
Could we set sell and buy to `0` and add a comment explaining what's going on? Otherwise I'm afraid this will be confusing in the future.

Reply:
Solved as per @sunce86 's comment.